gladier_kanzus/tools/gather_data.py

```diff
@@ -50,16 +49,34 @@ def ssx_gather_data(**data):
             int_filenames.append(filename)
             int_number, exp_name, beamline_run, int_index = int_match.groups()
             int_indices.append(int(int_index))
-        else:
-            # match idx-Kaleidoscope_15_00001_datablock.json
-            cbf_match = re.match(r'idx-\w+_\d+_(\d+)_datablock.json', filename)
-            if cbf_match:
-                cbf_index = int(cbf_match.groups()[0])
-                cbf_indices.append(cbf_index)
+        # else:
+        #     match idx-Kaleidoscope_15_00001_datablock.json
+        #     cbf_match = re.match(r'idx-\w+_\d+_(\d+)_datablock.json', filename)
+        #     cbf_match = re.match(r'idx-\w+_\d+_(\d+)_datablock.json', filename)
+        #     if cbf_match:
+        #         cbf_index = int(cbf_match.groups()[0])
+        #         cbf_indices.append(cbf_index)
+        if '.txt' in filename:
+            fpath = os.path.join(processing_dir,filename)
+            with open(fpath,'r') as f:
+                for line in f.readlines():
+                    match = re.findall(r"(\d+).cbf", line)
+                    if match:
+                        cbf_index = int(match[0])
+                        cbf_indices.append(cbf_index)
```

Comment:
Is this for dials v3? I'm wondering if the its are stored in this `.txt` file instead of as a separate `int-` file